src/app/static/package.json

```diff
@@ -29,14 +29,15 @@
     "babel-jest": "^24.8.0",
     "babel-loader": "^8.0.5",
     "bootstrap": "^4.3.1",
+    "bootstrap-vue": "^2.0.3",
     "d3-scale-chromatic": "^1.5.0",
     "form-data": "^2.5.1",
     "gulp": "^4.0.0",
     "gulp-clean-css": "^4.0.0",
     "gulp-rename": "^1.4.0",
     "gulp-sass": "^4.0.2",
     "ify-loader": "^1.1.0",
-    "jest": "^24.8.0",
+    "jest": "^25.0.0",
```

Comment:
I had to updated jest to a beta version because of a bug with jsdom that only just got fixed. Seems to be working ok.